ament_tools/build_types/cmake.py

```diff
@@ -167,8 +167,9 @@ def _common_cmake_on_build(self, should_run_configure, context, prefix, extra_cm
             solution_file = solution_file_exists_at(
                 context.build_space, context.package_manifest.name)
             cmd = prefix + [MSBUILD_EXECUTABLE]
-            if '-j1' in context.make_flags:
+            if not '-j1' in context.make_flags:
```

Comment:
linter will certainly complain about using `not <variable> in` instead of `<variable> not in`

Thanks @codebot you just made my life much easier!